Resources/Prototypes/Radium/Entities/Clothing/Head/hardsuit-helmets.yml

```
@@ -58,3 +58,17 @@
         Radiation: 0.2
+
+- type: entity
+  parent: [ BaseCentcommContraband, ClothingHeadHardsuitWithLightBase ] # Corvax-Resprite
+  id: ClothingHeadHelmetHardsuitVoxStealth
+  suffix: Vox
+  name: Vox Raiders helmet
+  description: A robust helmet for special operations.
+  components:
+  - type: BreathMask
+  - type: Sprite
+    sprite:  Radium/Clothing/Head/Hardsuits/helmstealth.rsi
```

yamllint (1.35.1)

[warning] too many spaces after colon

(colons)